subsys/net/ip/icmpv4.c

```diff
@@ -208,6 +208,7 @@ int net_icmpv4_send_echo_request(struct net_if *iface,
 
```

Comment:
In commit "net: ping: Improve the output of the ping function", I would instead name it "net: shell: Improve the output of the ping function" as this is more a shell improvement.

Reply:
Ok, reworded the commit message.